pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -659,10 +663,10 @@ func Convert_v1_Watchdog_To_api_Watchdog(source *v1.Watchdog, watchdog *Watchdog
 	return fmt.Errorf("watchdog %s can't be mapped, no watchdog type specified", source.Name)
 }
 
-func Convert_v1_Rng_To_api_Rng(source *v1.Rng, rng *Rng, _ *ConverterContext) error {
+func Convert_v1_Rng_To_api_Rng(c *ConverterContext, _ *v1.Rng, rng *Rng, _ *ConverterContext) error {
```

Comment:
You already have here a `*ConverterContext` as  a nameless parameter - last parameter in the signature. 

Shouldn't you just rename that one ? 

I'm OK with reordering the parameters, but not w/ duplicating them.

Reply:
Fixed. I missed that it was there already.